index.html

```diff
@@ -32,12 +28,154 @@ <h1 class="about-me">
 
         <p class="lets-connect"> LET'S CONNECT</p>
 
-        <ul class="buttons" >
+        <ul class="buttons">
           <li><a href="#"><img src="./images/twitter.png" alt="social"> </a></li>
           <li><a href="#"><img src="./images/Linkedin.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/medium.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/github.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
         </ul>
       </div>
-    </section>
\ No newline at end of file
+    </section>
+    <section class="work-section">
+      <div class="project-card">
```

Comment:
Please align the project details section with the image horizontally as it's in the design.
Yours:
![details](https://user-images.githubusercontent.com/35504149/137928476-01c379e4-3a10-4e00-bbbb-c4e680bb2e2c.png)
Orignal:
![orignal](https://user-images.githubusercontent.com/35504149/137928723-84c3bb8b-0bc3-4a03-88c7-1e5eeb01e196.png)

Reply:
Thanks for the correction @SajjadAhmad14